src/OrbitDB.js

```diff
@@ -226,11 +238,19 @@ class OrbitDB {
       onDrop: this._onDrop.bind(this),
       onLoad: this._onLoad.bind(this)
     })
+
+    const relayEvents = options.relayEvents ? options.relayEvents : this._relayEvents
```

Comment:
can be reduced to `const relayEvents = options.relayEvents || this._relayEvents` .
personally i would also move this closer to where it's used. i see you are trying to keep it by all the other option related stuffs.

Reply:
Done!